### ISSUE TITLE
Aligning high-resolution timer granularity to cross-origin isolated capability

### ISSUE THREAD

**Issue by yoavweiss** [mozilla/standards-positions#502]
## Request for Mozilla Position on an Emerging Web Specification

* Specification Title: HR-Time
* Specification or proposal URL: https://w3c.github.io/hr-time/#dfn-current-high-resolution-time
* Caniuse.com URL (optional): 
* Bugzilla URL (optional): 
* Mozillians who can provide input (optional): @annevk, @tomrittervg

### Other information
We recently [changed](https://github.com/w3c/hr-time/pull/93) the [HR-time spec](https://w3c.github.io/hr-time/) to better align its resolution clamping with [cross-origin isolated capability](https://html.spec.whatwg.org/multipage/webappapis.html#concept-settings-object-cross-origin-isolated-capability).

For Chromium, that means that it would be reducing its resolution in non-isolated contexts (regardless of the platform's site-isolation status) to 100 microseconds, and increasing it in cross-origin isolated contexts (even in platforms without site-isolation, e.g. Android) to 5 microseconds.

As y'all already clamp those timers to 1ms (AFAIK), I'd mostly like your position on the latter. 
Would y'all be interested in increasing timer granularity in contexts which have guarantees against pulling in cross-origin resources without their opt-in?

@annevk's [comments](https://github.com/w3c/hr-time/pull/93#issuecomment-704337546) and extremely-helpful review of the PR suggest you might, but I'm filing this to get an official position.

**Comment by annevk** [comment]
tl;dr: worth prototyping.

Yeah, this was implemented in https://bugzilla.mozilla.org/show_bug.cgi?id=1586761. We currently use 20μs in cross-origin isolated contexts and indeed use 1ms elsewhere. With a preference I think the latter might go even higher.

Great to see Chromium adopting this as well!
